src/chapters/interfaces-and-polymorphism/interfaces.rst

```diff
@@ -0,0 +1,285 @@
+Interfaces
+==========
+
+.. index:: ! interface
+
+An **interface** is a formal construction within Java. Interfaces allow us to create a set of behaviors that a class must fulfill.
+A Java interface may contain:
+
+#. Constants (that is, ``static final`` fields)
+#. Method signatures
+#. Static methods
+#. Default methods
+
+Creating an Interface
+---------------------
+
+Method Signatures
+^^^^^^^^^^^^^^^^^
+
+.. index:: ! method signature
+
+One really useful aspect of using an interface is the ability to specify method signatures.
+Recall that a **method signature** includes the name, parameters, and return
+type of a method, but no body.
+
+As part of our cat application, let's create a method signature, ``eat``, as part of an interface, ``Feedable``.
+
+.. admonition:: Example
+
+   .. sourcecode:: java
+      :linenos:
+
+      public interface Feedable
+      {
+
+         void eat();
+
+      }
+
+In the code above, you may have noticed few things:
+
+#. That we need to use the ``interface`` keyword to define our interface, ``Feedable``.
+#. ``eat`` only has a signature. We are NOT allowed to provide a body for methods defined in interfaces.
+#. ``eat`` also does not have an access modifier. Interface members are always ``public``, and while we may use the ``public`` modifier, it’s unnecessary. An interface method may NOT have an access modifier that is more restrictive than ``public``.
+#. The ``Feedable`` interface itself is declared ``public``, which means any other class may use it. We may also leave off ``public``, making the interface package-private, or usable only within the same package.
+#. The name is indicative of the behavior that the interface is intended to describe. While this is only a convention, most interfaces have names that are adjectives. Whatever you do, use meaningful names!
+
+Static Methods
+^^^^^^^^^^^^^^
+
+A static method in an interface can contain code in the body. However, a static method cannot contain any references to instance properties in other classes.
```

Comment:
This is good, but how about mentioning something about what static methods would be appropriate in an interface?  They should deal only with constants and universal formulas/behaviors, yes?

Reply:
Done! Thanks!